CondFormats/HGCalObjects/interface/HGCalCalibrationParameterSoA.h

```diff
@@ -0,0 +1,46 @@
+#ifndef CondFormats_HGCalObjects_interface_HGCalCalibrationParameterSoA_h
+#define CondFormats_HGCalObjects_interface_HGCalCalibrationParameterSoA_h
+
+#include <Eigen/Core>
+#include <Eigen/Dense>
+
+#include "DataFormats/SoATemplate/interface/SoACommon.h"
+#include "DataFormats/SoATemplate/interface/SoALayout.h"
+#include "DataFormats/SoATemplate/interface/SoAView.h"
+
+//#include "CondFormats/HGCalObjects/interface/HGCalMappingModuleIndexer.h"
+
+namespace hgcalrechit {
+
+  // human-readable data type to replace bool for memcpy of vector<>::.data()
+  // NOTE: bool and byte have the same memory size
+  using mybool = std::byte;
```

Comment:
I would suggest `char` or `unsigned char` as a replacement for `bool` instead of `std::byte`, because `std::byte b = true` and `if (b) { ... }` do not compile, while `unsigned char c = true` and if (c) { ... }` work fine.

Reply:
Or `uint8_t`.

Reply:
OK (unsigned char)